FEM/rf_mmp_new.cpp

```diff
@@ -181,7 +181,8 @@ bool MMPRead(std::string base_file_name)
     //----------------------------------------------------------------------
     // OK  MMPDelete();
     //----------------------------------------------------------------------
-    ScreenMessage("MMPRead ... ");;
+    ScreenMessage("MMPRead ... ");
+    ;
```

Comment:
A mistake from the previous implementation.  Would you please remove it?

Reply:
Done